src/pages/_document.tsx

```diff
@@ -0,0 +1,19 @@
+import { Head, Html, Main, NextScript } from 'next/document'
+
+import ThirdPartyScripts from 'src/components/ThirdPartyScripts'
+
+function Document() {
+  return (
+    <Html>
+      <Head>
+        <ThirdPartyScripts />
```

Comment:
We can move this to the _app.ts component and wrap it with next/head

Reply:
doing this will include more JS on the final bundle than needed

Reply:
Why? Since the next/head is already in use?

Reply:
https://partytown.builder.io/nextjs#configure

Reply:
Take a look here using the[ partytown inside the _app with next/head](https://github.com/vtex-sites/next.store/pull/8): the view source view-source:https://next-store-l8cbr9fj1-faststore.vercel.app/

Your branch view-source: view-source:https://sfj-b168bce--base.preview.vtex.app/

Reply:
This is the report with the <ThirdPartyScripts/> in _app vs on __document:
on _app:
```
Page                                       Size     First Load JS
┌ ○ /                                      2.62 kB         109 kB
├   /_app                                  0 B            97.3 kB
├ ● /[...slug]                             2.21 kB         113 kB
├ ● /[slug]/p                              4.56 kB         111 kB
├ ○ /404                                   431 B           103 kB
├ ○ /500                                   439 B           103 kB
├ ○ /account                               368 B           103 kB
├ λ /api/graphql                           0 B            97.3 kB
├ ○ /checkout                              374 B           103 kB
├ ○ /login                                 368 B           103 kB
└ ○ /s                                     2.15 kB         109 kB
+ First Load JS shared by all              97.3 kB
  ├ chunks/framework-fc97f3f1282ce3ed.js   44.9 kB
  ├ chunks/main-4cf47fefcd9ebea2.js        20.5 kB
  ├ chunks/pages/_app-7bc8dbb918c461c6.js  30.5 kB
  ├ chunks/webpack-566f5c6c8d77736b.js     1.55 kB
  └ css/7d73de50030d3a43.css               14.7 kB
```

on __document:
Page                                       Size     First Load JS
┌ ○ /                                      2.62 kB         110 kB
├   /_app                                  0 B             100 kB
├ ● /[...slug]                             2.21 kB         114 kB
├ ● /[slug]/p                              4.56 kB         112 kB
├ ○ /404                                   431 B           104 kB
├ ○ /500                                   439 B           104 kB
├ ○ /account                               368 B           104 kB
├ λ /api/graphql                           0 B             100 kB
├ ○ /checkout                              374 B           104 kB
├ ○ /login                                 368 B           104 kB
└ ○ /s                                     2.15 kB         110 kB
+ First Load JS shared by all              100 kB
  ├ chunks/framework-fc97f3f1282ce3ed.js   44.9 kB
  ├ chunks/main-4cf47fefcd9ebea2.js        20.5 kB
  ├ chunks/pages/_app-6d4dd2119ba131d9.js  33.2 kB
  ├ chunks/webpack-786ba62b2546136e.js     1.55 kB
  └ css/7d73de50030d3a43.css               14.7 kB

Reply:
Ok, thank you for the tests